client/ayon_maya/plugins/inventory/connect_ornatrix_rig.py

```diff
@@ -0,0 +1,135 @@
+        source_repre_id = source_container["representation"]
+        source_namespace = source_container["namespace"]
+
+        # Validate source representation is an alembic.
+        source_path = get_representation_path(
+            repre_contexts_by_id[source_repre_id]["representation"]
+        ).replace("\\", "/")
+        message = "Animation container \"{}\" is not an alembic:\n{}".format(
+            source_container["namespace"], source_path
+        )
+        if not source_path.endswith(".abc"):
+            self.display_warning(message)
+            return
+
+        ox_rig_containers = containers_by_product_type.get("oxrig")
+        print("containers", ox_rig_containers)
+        if not ox_rig_containers:
+            self.display_warning(
+                "Select at least one oxrig container"
+            )
+            return
+        source_nodes = []
+        for container in ox_rig_containers:
+            repre_id = container["representation"]
+            maya_file = get_representation_path(
+                repre_contexts_by_id[repre_id]["representation"]
+            )
+            _, ext = os.path.splitext(maya_file)
+            settings_file = None
+            if ext == ".zip":
+                settings_file = maya_file.replace(".oxg.zip", ".rigsettings")
+                print(settings_file)
+            else:
+                settings_file = maya_file.replace(ext, ".rigsettings")
+            if not os.path.exists(settings_file):
+                continue
+            with open(settings_file, "r") as fp:
+                source_nodes = json.load(fp)
+            grooms_file = None
+            if ext == ".zip":
+                grooms_file = maya_file
+            else:
+                grooms_file = maya_file.replace(ext, ".oxg.zip")
+            grooms_file = grooms_file.replace("\\", "/")
+            # Compare loaded connections to scene.
+            for node in source_nodes:
+                node_name = node.get("node").replace("|", "")
+                target_node = cmds.ls(f"{source_namespace}:{node_name}")
+                if not target_node:
+                    self.display_warning(
+                        "No target node found "
+                        "in \"animation\" or \"pointcache\"."
+                    )
+                    return
+                cmds.select(target_node)
+                mel.eval(f'OxLoadGroom -path "{grooms_file}";')
+
+    def display_warning(self, message, show_cancel=False):
+        """Show feedback to user.
+
+        Returns:
+            bool
+        """
+
+        from qtpy import QtWidgets
+
+        accept = QtWidgets.QMessageBox.Ok
+        if show_cancel:
+            buttons = accept | QtWidgets.QMessageBox.Cancel
+        else:
+            buttons = accept
+
+        state = QtWidgets.QMessageBox.warning(
+            None,
+            "",
+            message,
+            buttons=buttons,
+            defaultButton=accept
+        )
```

Comment:
Should this be parented to the Maya main window?

Reply:
I would prefer to align with other connecting actions in `inventory` folder and keeps it to `None`